internal/controllers/device_config_controller.go

```diff
@@ -494,6 +494,42 @@ func (d *DeviceConfigController) PatchHwConfigStatusByEthAddr(c *fiber.Ctx) erro
 	return d.PatchConfigStatusByEthAddr(c)
 }
 
+// PatchRuptelaConfigStatusByEthAddr godoc
+// @Description  Set what firmware was applied. None of the properties are required. Will not be set if not passed in. Endpoint is meant only for hardware devices self-reporting their template update.
+// @Tags         device-config
+// @Produce      json
+// @Success      200 "Successfully updated"
+// @Failure 500  "unable to parse request or storage failure"
+// @Param        ethAddr  path   string  true  "Ethereum Address"
+// @Param        config body DeviceRuptelaStatusPatch true "set any properties that were updated on the device"
```

Comment:
is this the same payload was what comes from mqtt topic ?

Reply:
yeah, the payload is like:
```
 {
    "sn": "R1-016528000016057",
    "battVolt": "622",
    "hwVersion": "FTX-04-12230",
    "imei": "016528000016057",
    "fwVersion": "00.06.61.53",
    "sigStrength": "30",
    "accessTech": "8",
    "operator": "310410",
    "locAreaCode": "33072",
    "cellId": "6875"
  }
```